www/~/%username/history/record-an-exchange.spt

```diff
@@ -78,7 +78,6 @@ if request.method == 'POST':
 {% extends "templates/profile.html" %}
 {% block content %}
 
-<h2>Record an Exchange</h2>
```

Comment:
We get an automatic `h1` from the `title` defined above. The `h2` here is redundant.